packages/bpk-svgs/readme.md

```diff
@@ -7,3 +7,19 @@
 ```sh
 npm install bpk-svgs --save-dev
 ```
+
+## Usage
+This component can be used on its own. The `BpkIcon` component for Web and Native will make it easier to use our icons.
```

Comment:
Actually, make that "...the icon components...."

Or remove it completely if Matt thinks that's best. He has more experience of what our consumers do with this information

Reply:
cc @matthewdavidson wdyt?

Reply:
```
## Usage

This package exposes Backpack SVG's in various formats:
- React components
  ``
  bpk-svgs/dist/js/icons/lg/*
  bpk-svgs/dist/js/icons/sm/*
  bpk-svgs/dist/js/spinners/*
  ``
- Sass variables containing datauri strings
  ``
  bpk-svgs/dist/scss/_elements.scss
  bpk-svgs/dist/scss/_icons-lg.scss
  bpk-svgs/dist/scss/_icons-no-color-lg.scss
  bpk-svgs/dist/scss/_icons-no-color-sm.scss
  bpk-svgs/dist/scss/_icons-sm.scss
  bpk-svgs/dist/scss/_spinners.scss
  ``
- An icon font
  ``
  bpk-svgs/dist/font/BpkIcon.ttf
  ``
```

Reply:
This is just some useful information for folk that hit https://www.npmjs.com/package/bpk-svgs directly. 

We don't actually expose this info anywhere on our docs site so we don't need to think too much about it.

Reply:
👍